email_alert_service/config.rb

```diff
@@ -16,18 +16,22 @@ def app_root
     end
 
     def rabbitmq
-      all_configs = YAML.load(ERB.new(File.read(app_root+"config/rabbitmq.yml")).result)
+      # rubocop:disable Security/YAMLLoad
+      all_configs = YAML.load(ERB.new(File.read(app_root + "config/rabbitmq.yml")).result)
```

Comment:
Can we use `safe_load` here instead of disabling the warning?

Reply:
I tried using `safe_load` but that stops `defaults: &defaults` and `<<: *defaults` in the YAML from working since they are dynamic features.

Reply:
Interesting, after some digging you can enable them via the rather clunky [`YAML.safe_load(input, [], [], true)`](http://ruby-doc.org/stdlib-2.1.0/libdoc/psych/rdoc/Psych.html#method-c-safe_load)